src/lib.rs

```diff
@@ -195,6 +195,33 @@ let result = nad_ft_to_m.convert(my_point).unwrap();
 assert_relative_eq!(result.x(), 1450880.29, epsilon=1e-2);
 assert_relative_eq!(result.y(), 1141263.01, epsilon=1e-2);
 ```
+
+You can also transform entire geometries from `geo-types` by using the
+`Transform` trait.
+
+```
+use proj::{Proj, Transform};
+use geo_types::{Coordinate, line_string};
+
+let line = line_string![
+    (x: -116.590457069172_f64, y: 32.55730630167689),
+    (x: -116.590411068973, y: 32.55714830169309),
+];
+let proj = Proj::new_known_crs("EPSG:4326", "EPSG:6366", None).unwrap();
+
+// create a new line with a different projection
+let new_line = line.transformed(&proj).unwrap();
+
+assert_eq!(new_line[0], Coordinate { x: 538447.8454476658, y: 3602285.563945497, });
```

Comment:
Please use `assert_relative_eq` instead of `assert_eq!` it's not uncommon to see these values shift a small amount between proj releases or across platforms.

Reply:
fixed